service/core/kafka/client/src/main/scala/com/lightbend/lagom/internal/broker/kafka/KafkaSubscriberActor.scala

```diff
@@ -143,7 +139,14 @@ private[lagom] class KafkaSubscriberActor[Payload, SubscriberPayload](
       FlowShape(unzip.in, committer.out)
     })
 
-    pairedCommittableSource.via(committOffsetFlow)
+      ReactiveConsumer
+        .committablePartitionedSource(consumerSettingsWithUri, subscription)
+          .flatMapMerge(1000 /* what to put in here? */, { case (_, source) =>
```

Comment:
That needs to be more than the maximum number of partitions to be expected for this subscriber, otherwise some partitions will be starved.